src/wallet/wallet.cpp

```diff
@@ -2034,7 +2034,7 @@ std::optional<libzcash::AccountId> CWallet::FindAccountForSelector(const ZTXOSel
 }
 
 std::optional<AddressUFVKMetadata>
-CWallet::UFVKMetadataForAddress(const CTxDestination& address) const {
```

Comment:
This should probably be a method on the keystore, rather than declared here. There are changes that I would love to try to backport that make it so that the CWallet class uses composition for its relationship to the keystore, rather than inheritance.

Reply:
See #6463.